libs/cardano-protocol-tpraos/CHANGELOG.md

```diff
@@ -102,10 +103,10 @@
 * Add `mkBHeader`
 * Move from `cardano-ledger-shelley-test`: `mkOCert`, `mkBHBody`, `mkBlock`
 * Move `AllIssuerKeys` from `cardano-ledegr-shelley-test`. Rename its fields:
-  *  `cold` - > `aikCold`
-  *  `hot` - > `aikHot`
-  *  `vrf` - > `aikVrf`
-  *  `hk` - > `aikColdKeyHash`
+  - `cold` - > `aikCold`
+  - `hot` - > `aikHot`
+  - `vrf` - > `aikVrf`
+  - `hk` - > `aikColdKeyHash`
```

Comment:
Are these having spaces between the `-` (hyphen) and the `>` (right angle-bracket) characters?